sbol2/property.py

```diff
@@ -223,6 +223,42 @@ def __str__(self):
         else:
             return str(self._sbol_owner.properties[self._rdf_type])
 
+    @staticmethod
+    def guess_args(validators, initial_value):
```

Comment:
Probably the easiest way to deal with this problem for now.  Alternatively, we could try a more formal approach such as [dispatch generic functions](https://www.python.org/dev/peps/pep-0443/), but it's probably not worth the extra cycles.

In pySBOL3 we can mostly eliminate signature overloading and just use specialized methods.  However, there is at least one fundamental case where I think it is desirable to retain overloaded signatures.  This might be a good place to use the generic dispatch formalism above.

Reply:
I'm curious what that one fundamental case is. Can you open an issue at https://github.com/SynBioDex/pySBOL3 for it?